app/src/public/css/login-publishing.css

```
@@ -0,0 +1,168 @@
+* {
+    box-sizing: border;
+    color: #c4c4c4;
+    font-family: "Noto Sans KR", sans-serif;
+}
+.wholeWrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* background-color: rgb(226, 226, 226); */
+    width: 100vw;
+    height: 100vh;
```

하위 요소들을 중앙 정렬하려고 넓이랑 높이를 100vw vh주신거 같은데
그냥 자기 자신에 margin 0 auto해서 가운데 정렬 할 수도 있어요
```suggestion
   margin: 0 auto;
```

```
width: fit-content;
height: fit-content;
// 이렇게 하면 하위 요소에 맞춰서 넓이랑 높이가 맞춰지니까 알아두시면 좋을거 같습니다
```